### ISSUE TITLE
Edge labels are not displayed at the initialization of the diagram

### ISSUE THREAD

**Issue by hamzaed** [xatkit-bot-platform/xatkit-eclipse#41]
To solve #34 we used the Java service mechanism provided by Sirius and reused our [utils Java methods](https://github.com/xatkit-bot-platform/xatkit-runtime/blob/e45161b90eb5763f7fda6942e21e1c22c0eb54e5/core/src/main/java/com/xatkit/util/ExecutionModelUtils.java) to get the intent name. However, after adding the ELK extension and integrating an ELK algorithm to improve the layout the diagram, we noticed that labels are not displayed after initializing the diagram. We need to click on refresh to display them then auto layout to arrange the diagram.

I think it's a bug in the Sirius ELK implementation.

Quick fix: We need to write the expression in AQL rather than Java (which might be complicated)

**Comment by jcabot** [comment]
Did you open the bug in the sirius or elk repos?

**Comment by hamzaed** [comment]
The following expression seems to work (and it's compatible with the autolayout). @gdaniel what do you think?: 
```
aql:self.eAllContents(xbase::XFeatureCall) 
	-> asSequence() 
	-> collect(e | e.oclAsType(xbase::XFeatureCall).feature)  
	-> filter(types::JvmGenericType) 
	-> select(s | s.oclAsType(types::JvmGenericType).members
		-> asSequence()
		-> exists(m | m.simpleName.equals('base')))
	-> first().identifier
```
@jcabot I didn't open an issue because I wanted the be sure of the origin of the error (maybe it's our service).

**Comment by hamzaed** [comment]
At the end the problem was not related to ELK algorithms (the labels are not displayed even using the built-in auto-layout feature). 

After some tests, I discovered that the problem occurs because I was calling the Java service (which returns a set of `EventDefinition`) in an AQL expression (which in theory should work as described in the [documentation](https://www.eclipse.org/sirius/doc/specifier/general/Writing_Queries.html#service_navigation)). Using AQL, I refine the result to get the name of the event definition. 

I was able to fix the issue by updating the Java service to return directly the name of the event and call the service using `service:` instead of the mixed mode using `aql:`.
Both this solution and the solution described in the comment above work but I prefer this one since it relies on the code that has been validated and that is used in the other plugins.
